lib/components/user/monitored-trip/route-block-wrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentContext } from '../../../util/contexts'
+import { Itinerary } from '@opentripplanner/types'
+import React, { useContext } from 'react'
+
+interface Props {
+  itinerary: Itinerary
+}
+const RouteBlockWrapper = ({ itinerary }: Props) => {
```

Comment:
This is causing problems with the old route blocks, but I think that's fine. <img width="721" alt="Screenshot 2024-04-11 at 1 01 54 PM" src="https://github.com/opentripplanner/otp-react-redux/assets/86619099/3c1b14a1-b00b-4c35-900c-25dba5c3abe8">